src/mrsimulator/simulator/__init__.py

```diff
@@ -132,7 +133,8 @@ class Simulator(Parseable):
     spin_systems: List[SpinSystem] = []
     methods: List[Method] = []
     config: ConfigSimulator = ConfigSimulator()
-    # indexes = []
+    _precomputed_pathways: list = PrivateAttr([])
+    _precomputed_weights: list = PrivateAttr([])
```

Comment:
Might be better to have a separate object for this. not a member of simulator object. See comment below